.rubocop.yml

```diff
@@ -30,3 +30,6 @@ Metrics/ClassLength:
 Performance/FixedSize:
   Exclude:
     - 'test/**/*'
+
+Style/AndOr:
+  EnforcedStyle: conditionals
```

Comment:
By default this is `always` which completely forbids usage of `and` and `or`. Thoughts on `always` vs `conditionals`?